dotcom-rendering/src/amp/lib/real-time-config.ts

```diff
@@ -75,7 +75,7 @@ export const realTimeConfig = ({
 		notUndefined,
 	);
 
-	const options = timeoutMillis ? { timeoutMillis } : {};
+	const options = timeoutMillis !== undefined ? { timeoutMillis } : {};
```

Comment:
Maybe simpler to just do:

```suggestion
	const options = { timeoutMillis };
```

Reply:
This code has since been removed :)